consensus/src/fallback.rs

```diff
@@ -156,6 +171,9 @@ impl Fallback {
         if block.fallback == 0 {
```

Comment:
make two functions instead of having all the code here: If block.fallback { handle_vote_common} else {handle_vote_fallback}.

Also why you have {0,1} and not a bool?